src/machines/java/com/enderio/machines/common/config/client/MachinesClientConfig.java

```diff
@@ -6,8 +6,13 @@ public class MachinesClientConfig {
 
     public static MachinesClientBlocksConfig BLOCKS;
 
+    public final ForgeConfigSpec.ConfigValue<Float> IO_CONFIG_NEIGHBOUR_TRANSPARENCY;
+
     public MachinesClientConfig(ForgeConfigSpec.Builder builder) {
         BLOCKS = new MachinesClientBlocksConfig(builder);
 
+        builder.push("ioconfig");
+        IO_CONFIG_NEIGHBOUR_TRANSPARENCY = builder.comment("Neighbour Transparency [0-1]").define("machineParticles", 0.4F);
```

Comment:
machineParticles seems the wrong string for this no?

Reply:
Yep, that is a mistake. (shouldnt blindly copy paste)